0miner

```diff
@@ -185,10 +191,12 @@ fi
 if [[ ${!xversion} != "" && -f ${mpath}//"${!xversion}"/nvoc-miner.json ]]
 then
   HCD="${mpath}/"${!xversion}"/$(jq -r .hcd ${mpath}/"${!xversion}"/nvoc-miner.json)"
+  echo $LAUNCH $HCD
   eval $LAUNCH $HCD
 elif [[ -f ${mpath}/nvoc-miner.json ]]
 then
   HCD="${mpath}/$(jq -r .hcd ${mpath}/nvoc-miner.json)"
+  echo $LAUNCH $HCD
   eval $LAUNCH $HCD
 
 
```

Comment:
Add a warning here that explains the following code is deprecated and is not used anymore if a pluggable miner gets matched.